pages/common/ls.md

```diff
@@ -34,3 +34,7 @@
 - Only list directories:
 
 `ls -d {{*/}}`
+
+- List all files with no stat, sort, color nor any slowdown (useful when listing huge number of files)
```

Comment:
```suggestion
- List all files with no stat, sort, color nor any slowdown
```

Reply:
I don't agree with removing this parenthetical statement.  How about using "large" instead of "huge":
```suggestion
- List all files with no stat, sort, color nor any slowdown (useful when listing large numbers of files)
```